src/containers/ConversionForm/index.tsx

```diff
@@ -36,6 +38,14 @@ interface ConversionProps {
     setError: ActionCreator<SetErrorAction>;
 }
 
+const validFileExtensions: ExtensionMap = {
+    // TODO: update this with correct extensions
+    Smoldyn: "txt",
+    cytosim: "txt",
+    cellPACK: "txt",
+    SpringSaLaD: "txt",
+};
+
 const selectOptions = Object.keys(AvailableEngines).map(
```

Comment:
Could be nice to add an extension hint (.txt) after the engine names in the future?